src/filters/index.js

```diff
@@ -1,55 +1,68 @@
 // @ts-check
+
 /**
- * @param {string} str
- * @return {string}
+ * Uppercase the first character of a string.
+ *
+ * @param  {string} str String to uppercase the first letter of
+ * @return {string}     Original string with the first letter in uppercase
  */
 export const capitalize = str => str[0].toUpperCase() + str.slice(1);
 
 /**
- * @param {number} str
- * @param {string} currency
- * @param {string} [locale = navigator.language]
- * @return {string}
+ * Formats a number as currency using the user's locale. 1234.56 => '$1,234.56'
+ *
+ * @param  {number} num      The number to be formatted
+ * @param  {string} currency The type of currency, such as 'USD' or 'JPY'
+ * @param  {string} locale   The locale for formatting the number, such as 'en-US'. Defaults to navigator.language
+ * @return {string}          The number as a string formatted in the locale of the user
  */
-export function currency(str, currency, locale = navigator.language) {
+export function currency(num, currency, locale = navigator.language) {
   // Alternative: (73.57).toLocaleString('de-DE',{style:'currency',currency:'EUR'});
   return new Intl.NumberFormat(locale, {
     style: 'currency',
     currency,
-  }).format(str);
+  }).format(num);
 }
 
 /**
- * @param {number} str
- * @param {string} [locale = navigator.language]
- * @return {string}
+ * Formats a number to a culture code locale. 1234 => '1,234'
+ *
+ * @param  {number} num    The number to be formatted
```

Comment:
Same as above. If we get type checking working inside a Vue template, then I think this will error when receiving a string.